roles/openshift_health_checker/test/openshift_check_test.py

```diff
@@ -94,3 +95,22 @@ def test_get_var_error(task_vars, missing_keys):
 def test_get_var_default(task_vars, missing_keys):
     default = object()
     assert dummy_check(task_vars).get_var(*missing_keys, default=default) == default
+
+
+@pytest.mark.parametrize("keys, convert, expected", [
+    (("foo",), str, "42"),
+    (("foo",), float, 42.0),
+    (("bar", "baz"), bool, False),
+])
+def test_get_var_convert(task_vars, keys, convert, expected):
+    assert dummy_check(task_vars).get_var(*keys, type=convert) == expected
+
+
+@pytest.mark.parametrize("keys, convert", [
+    (("foo",), "bogus"),
+    (("bar", "baz"), "int"),
+    (("bar.baz"), "float"),
```

Comment:
Funny that this test uses all strings, and the other uses all functions as the `convert` argument. I think dropping strings altogether makes the design and usage simpler.